### PR TITLE
map cutout

### PR DESCRIPTION
**Pull request by TylerGrantSmith**
I attempted to back out all of those commits and rebase, and that seemed like a huge hassle with tons of repetitive conflicts.  Decided it would be just easier to delete that branch and start with a fresh branch.

### CODE REVIEW

**Comment by TylerGrantSmith** [comment]
added the map simplification referenced in https://github.com/kasaai/pc-pricing-tutorial/issues/69#issuecomment-448346554 .  Should help with the load time at the expense of negligible map detail.

File: analysis/map.R [hunk 1]
```diff
@@ -8,29 +8,58 @@ exposures_by_hasc <- risks_table_mapped %>%
...
+
 brazil <- bra_layer1 %>%
   left_join(exposures_by_hasc, by = c(HASC_1 = "hasc"))
 
 bins <- c(0, 10^3,10^4, 10^5, 10^6, Inf)
 pal <- colorBin("YlOrRd", domain = brazil$exposures, bins = bins)
 
+bra_bbox <- sf::st_bbox(bra_layer1) %>% unname
+
+earth <-
```
Comment (kevinykuo) [inline analysis/map.R]: Could we move the `sf::st_polygon()` call to be on the same line as `earth`, get rid of `x = `, and put the `list()` on a new line? See https://style.tidyverse.org/syntax.html#long-lines.

File: analysis/map.R [hunk 2]
```diff
@@ -8,29 +8,58 @@ exposures_by_hasc <- risks_table_mapped %>%
...
+bra_layer0 <- bra_layer0 %>% rmapshaper::ms_simplify()
+bra_layer1 <- bra_layer1 %>% rmapshaper::ms_simplify()
+
 brazil <- bra_layer1 %>%
   left_join(exposures_by_hasc, by = c(HASC_1 = "hasc"))
 
 bins <- c(0, 10^3,10^4, 10^5, 10^6, Inf)
 pal <- colorBin("YlOrRd", domain = brazil$exposures, bins = bins)
 
+bra_bbox <- sf::st_bbox(bra_layer1) %>% unname
```
Comment (kevinykuo) [inline analysis/map.R]:
```suggestion
bra_bbox <- sf::st_bbox(bra_layer1) %>% unname()
```

See https://style.tidyverse.org/pipes.html#no-arguments.

File: analysis/map.R (hunk 1 above)
Comment (kevinykuo) [reply analysis/map.R]: Actually, let's do:
```
earth <- sf::st_polygon(list(rbind(
  c(360, -360),
  c(360, 360),
  c(-360, 360),
  c(-360, -360),
  c(360, -360)
))) %>%
  sf::st_sfc(crs = sf::st_crs(bra_layer0), check_ring_dir = T) %>%
  sf::st_sf()
```
TIL about https://github.com/r-lib/styler :)

**Pull request merged**